src/steamship/agents/logging.py

```diff
@@ -37,3 +42,34 @@ class AgentLogging:
     AGENT = "agent"
     TOOL = "tool"
     SYSTEM = "system"
+    LLM = "llm"
+
+
+LOGGING_FORMAT = {
+    "level": "%(levelname)s",
+    "host": "%(hostname)s",
+    "where": "%(module)s.%(filename)s.%(funcName)s:%(lineno)s",
+    "type": "%(levelname)s",
+    "stack_trace": "%(exc_text)s",
+    "message_type": "%(message_type)s",
+    "component_name": "%(component_name)s",
+    AgentLogging.IS_MESSAGE: f"%({AgentLogging.IS_MESSAGE})s",  # b doesn't work. Unsure how to make a bool
```

Comment:
Naieve response is the way you probably were avoiding because it felt silly: `'true' if AgentLogging.IS_MESSAGE else 'false'`